### ISSUE TITLE
SP Starter Kit - Hyperlink field not resolving value like {site}/SiteAssets/contoso_sitelogo.png

### ISSUE THREAD

**Issue by pk091990** [SharePoint/sp-dev-docs#4355]
This issue is related to provisioning of [SP Starter KIt](https://github.com/SharePoint/sp-starter-kit/tree/master/provisioning)
In the starterkit.xml, Hyperlink field i.e. "PnPPortalLinkUrl" value is passing like a normal url i.e. https:\

![image](https://user-images.githubusercontent.com/46436213/61782154-32bf5880-ae23-11e9-9f67-4267733acaad.png)

I need to pass a image url from site ->Site Assets folder and page url from site-> Site Pages in PnPPortalLinkUrl field.
Earlier I used to pass like in {site}/SiteAssets/contoso_sitelogo.png format. However this is not working now.

![image](https://user-images.githubusercontent.com/46436213/61782168-39e66680-ae23-11e9-850d-5ef86009b459.png)

![image](https://user-images.githubusercontent.com/46436213/61782188-423ea180-ae23-11e9-854c-1f7f6dd9f6ee.png)

I've tried with multiple ways like {site}, {sitecollection}, {sitecollection}/SiteAssets/... but kit is just skipping that Datarow . Is there any change from your end or anyother way to implement it as it was working earlier. Please help !

**Comment by bcameron1231** [comment]
Thanks for the question. However, this is a Starter Kit question and should be in the PnP Starter Kit Github Repo https://github.com/SharePoint/sp-starter-kit/issues. Looks like you may have already asked the question over there https://github.com/SharePoint/sp-starter-kit/issues/309

**Comment by pk091990** [comment]
Yes, the issue is in our production, So just asked on both platform for quick response :)

**Comment by bcameron1231** [comment]
I can understand the concern, however, this ticket will likely get moved over to the Starter Kit repository. Just wanted to give you a heads up, to try and keep Starter Kit questions over there, and General SP Development questions here :)
